src/utils.ts

```diff
@@ -35,11 +35,19 @@ export const Logger = {
     console.warn(sanitizeMsg(message));
   },
   error(message) {
-    console.error(sanitizeMsg(message));
+    if (typeof message === 'string') {
```

Comment:
can you please apply this logic to all log levels in the logger?

Reply:
will update

Reply:
edit: to fix codeql warnings, instead of checking type, i figured it would be better to just convert the message into a string. this keeps the sanitation of the message and prevents log injection as opposed to the previous commit.